src/api/burn/test/BalUtilUnitTest/BalUtilUnitTest.vcxproj

```diff
@@ -25,6 +25,7 @@
     <CharacterSet>Unicode</CharacterSet>
     <CLRSupport>true</CLRSupport>
     <SignOutput>false</SignOutput>
+    <IsWixTestProject>true</IsWixTestProject>
```

Comment:
It seems like this can be in `internal\WixBuildTools.TestSupport.Native\build\WixBuildTools.TestSupport.Native.targets` instead of putting it in every project.

Reply:
I thought about that but thought it was good to see in the test project. Still considering using this property to set all things "test-related" (like no signing) but not sure that's a good idea. Undecided.